docs/diagrams/CommitActivityDiagram.puml

```diff
@@ -11,7 +11,7 @@ start
 if () then ([command commits CcaCommander])
```

Comment:
Would it be better if there are line breaks here?
```suggestion
if () then ([command \ncommits \nCcaCommander]\n)
```
It would look something like this
<img width="266" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-1/tp/assets/56021409/fc6ab399-2e46-4fb9-9b03-246553e5239e">
instead of 
<img width="345" alt="image" src="https://github.com/AY2324S1-CS2103T-F11-1/tp/assets/56021409/f5237245-876d-47ce-a435-3a805622541b">

Reply:
Agreed, will make the change!